### PR TITLE
More cleanup of the individual pages, use the extra space better

### PR DESCRIPTION
**Pull request by jwforres**
Consistent placing of created date
Adds mounts to the pod template on pages other than overview
Make volumes use the correct api format to show more info

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -763,3 +763,11 @@ labels + dl {
   margin-top: 21px;
 }
 
+h3 + hr {
```
Comment (spadgett) [inline assets/app/styles/_core.less]: I wonder if h3 + hr shouldn't just be an h3 bottom border on these pages so we always have consistent header styles and don't have to add hr to the markup. Maybe have a class for each of these pages, `.resource-details`, and a `.resource-details h3` style?

File: assets/app/views/_pod-template.html [hunk 2]
```diff
@@ -72,6 +72,19 @@
         </div>
       </div> 
 
+      <div row ng-if="detailed" ng-repeat="mount in container.volumeMounts" class="icon-row">
+        <div>
+          <!-- For some reason the DB icon is massive compared to the others -->
+          <span aria-hidden="true" style="font-size:13px;line-height:normal" class="fa fa-database"></span>
```
Comment (spadgett) [inline assets/app/views/_pod-template.html]: Hm, I was able to disable this style checking out your pull request, and it looked OK in Chrome for me.

File: assets/app/views/_pod-template.html [hunk 3]
```diff
@@ -72,6 +72,19 @@
         </div>
       </div> 
 
+      <div row ng-if="detailed" ng-repeat="mount in container.volumeMounts" class="icon-row">
```
Comment (spadgett) [inline assets/app/views/_pod-template.html]: It's really nice to see the mounts in the UI now.

File: assets/app/views/browse/_pod-details.html [hunk 4]
```diff
@@ -1,64 +1,102 @@
...
+          <div ng-if="volume.persistentVolumeClaim">
+            <dt>Type:</dt>
+            <dd>persistent volume claim</dd>
+            <dt>Claim name:</dt>
+            <dd>{{volume.persistentVolumeClaim.claimName}}</dd>
+            <dt>Read only:</dt>
+            <dd>
+              <span ng-if="volume.persistentVolumeClaim.readOnly">yes</span>
+              <span ng-if="!volume.persistentVolumeClaim.readOnly">no (write allowed)</span>
+            </dd>
```
Comment (spadgett) [inline assets/app/views/browse/_pod-details.html]: Mode: Read-only or Mode: Read-write?

File: assets/app/views/browse/build-config.html [hunk 5]
```diff
@@ -5,137 +5,141 @@
...
-              </a>
-            </dt>
-            <dd>
-              <code>oc start-build {{buildConfig.metadata.name}} -n {{project.metadata.name}}</code>
-              <copy-to-clipboard-button clipboard-text="'oc start-build ' + buildConfig.metadata.name + ' -n ' + project.metadata.name"></copy-to-clipboard-button>
-            </dd>
+              <dt>Manual:</dt>
+              <dd>
+                <span>
+                  <button class="btn btn-default btn-xs" ng-click="startBuild(buildConfig.metadata.name)" ng-disabled="buildConfig.metadata.deletionTimestamp || (buildConfigBuildsInProgress[buildConfig.metadata.name] | hashSize) > 0">Build</button>
```
Comment (spadgett) [inline assets/app/views/browse/build-config.html]: Apply the same button style to the individual build page?

**Comment by spadgett** [comment]
These pages look a lot better. Most of my comments are nits, otherwise LGTM.

File: assets/app/views/_pod-template.html (hunk 2 above)
Comment (jwforres) [inline assets/app/views/_pod-template.html]: so strange, I can't recreate what i was seeing yesterday

**Comment by jwforres** [comment]
[merge]

**Pull request merged**